common/djangoapps/student/management/commands/manage_group.py

```diff
@@ -103,7 +103,8 @@ def _resolve_permissions(self, permissions):  # lint-amnesty, pylint: disable=mi
             except LookupError as exc:
                 raise CommandError(str(exc))  # lint-amnesty, pylint: disable=raise-missing-from
 
-            content_type = ContentType.objects.get_for_model(model_class)
+            # Fetch content type for model, including proxy models.
+            content_type = ContentType.objects.get_for_model(model_class, for_concrete_model=False)
```

Comment:
Good to know!

Reply:
Yes - it's definitely a refactor. I'm still getting used to the conventional commit msgs! Changed.